forgettable-frontend/src/components/CompanyCard/CompanyCard.js

```diff
@@ -0,0 +1,156 @@
+import {Avatar, AvatarGroup, StyledEngineProvider, SvgIcon} from '@mui/material';
+import React from 'react';
```

Comment:
I'm not sure what convention we're following but the convention I've seen so far is to import useState and other imports like 
import { useState. useEffect } from 'react';